neil

```diff
@@ -1,852 +1,14 @@
 #!/usr/bin/env bb
 ;; -*- mode: clojure -*-
 
-;; DO NOT EDIT DIRECTLY. GENERATED FROM scr/babashka/neil.clj
+(def deps
+ '{org.babashka/neil {:git/url "https://github.com/babashka/neil.git"
+                      :git/tag "v0.1.41" :git/sha "77288d4"}
+   org.babashka/spec.alpha {:git/url "https://github.com/babashka/spec.alpha"
+                            :git/sha "1a841c4cc1d4f6dab7505a98ed2d532dd9d56b78"}})
```

Comment:
I had to put the `org.babashka/spec.alpha` dependency here or else the `clj -M:neil` test would fail.